### ISSUE TITLE
benefits of bonito over guppy

### ISSUE THREAD

**Issue by mehdiborji** [nanoporetech/bonito#226]
I am new to basecalling. Currently we have some rudimentary basecalling setup which uses guppy and CPUs. I have gained access to a V100 and I am trying to get the most of our fantastic Nanopore data! I am wondering if it makes sense to use bonito over more established guppy. It seems I can do model training with bonito which sounds very promising. Also it seems with latest versions of bonito, it's as performant as guppy https://github.com/nanoporetech/bonito/issues/57. It's also easier to install, at least for me! So my question is what are the possible benefits that I might lose when using bonito instead of guppy?

A second important question for me is how I can use model training to my advantage. For example if I provide high-quality reads and read identifiers (10X style adapter-barcode-umi-TSO sequences) instead of actual genome as reference for model training would it help detecting barcodes in my reads?

**Comment by iiSeymour** [comment]
Hey @mehdiborji 

Switching from CPU calling to GPU calling with a V100 is great, you should see much higher calling speeds! Yes, Bonito provides model training and should be on par with Guppy in terms of speed for the most part. With regards to training from none genomic references, this is absolutely possible, and fine-tuning one of the provided pre-trained models is probably a good starting point.
